extensions/ql-vscode/package.json

```
@@ -58,15 +53,6 @@
     "onCommand:codeQL.chooseDatabaseInternet",
     "onCommand:codeQL.chooseDatabaseGithub",
     "onCommand:codeQLDatabases.chooseDatabase",
```

Agree with @robertbrignull that `onCommand:codeQLDatabases.chooseDatabase` can be removed, since it doesn't exist 😄 

```suggestion
    "onCommand:codeQL.chooseDatabaseGithub",
```

---

Yes, good catch @robertbrignull! :)